automatic/nsis.install/legal/LICENSE.txt

```diff
@@ -0,0 +1,115 @@
+Applicable licenses
```

Comment:
The header needs to be different: 
```suggestion
From: <INSERT URL HERE!>

Applicable licenses
```

Reply:
Done.

I thought I read some discussion about this not being required anymore if the URL is in the verification, but I can't find it.